src/uptane/initialize.cc

```diff
@@ -5,13 +5,13 @@
 
 #include "bootstrap.h"
 #include "logging.h"
+#include "uptane/cryptokey.h"
 #include "uptane/uptanerepository.h"
 
 namespace Uptane {
 
 // Postcondition: device_id is in the storage
-bool Repository::initDeviceId(const ProvisionConfig& provision_config, const UptaneConfig& uptane_config,
-                              const TlsConfig& tls_config) {
+bool Repository::initDeviceId(const ProvisionConfig& provision_config, const UptaneConfig& uptane_config) {
```

Comment:
Not relevant to your changes, but we are we bothering passing in UptaneConfig if we have a reference to the entire Config object in the Repository class already?

Reply:
Yes, It looks strange, I don't understand this to. You may ask it Anton maybe he can answer.